### PR TITLE
THE BASELINE

### PR DESCRIPTION
**Pull request by pchampio**
Use of `DataLoader`, finish the training, add eval.

### CODE REVIEW

**Comment by rebouvet** [comment]
Je regarde ce soir ce que tu as fais mais ça à l'air propre !

**Comment by pchampio** [comment]
Tu veux finir la baseline ? (reprendre ce qui est fait et ajouter "DataLoader, finish the training, add eval" )

**Comment by rebouvet** [comment]
Oui, je regarde ça ce soir :+1:

**Comment by rebouvet** [comment]
Bon j'ai pas mal tourné en rond.
J'ai mis du temps à comprendre le problème du RNN+batch avec des vecteurs de tailles variables pour finalement revenir à une version très similaire à la tienne.

Je viens de trouver une ressource qui peu être utile : 
https://towardsdatascience.com/taming-lstms-variable-sized-mini-batches-and-why-pytorch-is-good-for-your-health-61d35642972e

Si on ne fait pas de padding on ne pourra pas faire de batch je pense.
Je pense regarder demain comment intégrer ça a part si vous avez d'autres idées.
